eng/testing/linker/SupportFiles/Directory.Build.targets

```diff
@@ -19,6 +19,17 @@
     </ItemGroup>
   </Target>
 
+  <!-- This will need to change to AfterTargets=PrepareForILLink when we get a new SDK -->
```

Comment:
Will this cause a failure when we ingest the new SDK?

Reply:
Yes. :( @joperezr and I were discussing on Friday that we need to make the infrastructure team aware that when a new SDK comes, there has been a few breaking changes in the linker that we will need to fix up.

Cc @sbomer